### PR TITLE
Blank jobs

### PR DESCRIPTION
**Pull request by g4s8**
#760 - add jobs to `blanks.xml`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#977](https://github.com/zerocracy/farm/issues/977) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#977](https://github.com/zerocracy/farm/issues/977) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1180665787); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

File: src/main/resources/com/zerocracy/stk/pm/in/orders/close_job.groovy [hunk 1]
```diff
@@ -39,41 +40,41 @@ import com.zerocracy.radars.github.Job
...
         .token("job;${job}")
         .param(
-          'message',
-          new Par(
-            '@%s the issue is closed not by @%s (its creator);',
-            'I won\'t close the order;',
-            'please, re-open it and ask @%2$s to close it'
-          ).say(claim.author(), issue.author().login())
-        )
+        'message',
```
Comment (carlosmiranda) [inline src/main/resources/com/zerocracy/stk/pm/in/orders/close_job.groovy]: @g4s8 why change the indentation?

File: src/main/resources/com/zerocracy/stk/pm/in/orders/close_job.groovy [hunk 2]
```diff
@@ -39,41 +40,41 @@ import com.zerocracy.radars.github.Job
...
   ClaimIn claim = new ClaimIn(xml)
   String job = claim.param('job')
   Wbs wbs = new Wbs(project).bootstrap()
+  Issue.Smart issue = new Issue.Smart(new Job.Issue(github, job))
+  String author = issue.author().login().toLowerCase(Locale.ENGLISH)
   if (!wbs.exists(job)) {
-    // @todo #536:30min Every time we close a ticket in GitHub, which was not a job,
-    //  or when we close a pull request, which was not merged, we should
-    //  add it to blanks.xml
+    String kind = issue.pull ? 'pull-request' : 'issue'
```
Comment (carlosmiranda) [inline src/main/resources/com/zerocracy/stk/pm/in/orders/close_job.groovy]: @g4s8 can we inline this statement? Doesn't seem to be used anywhere else.

**Comment by g4s8** [comment]
@carlosmiranda fixed in 256015e

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by carlosmiranda** [comment]
@yegor256 ping

**Comment by g4s8** [comment]
@yegor256 Rultor failed to push: `error: failed to push some refs to 'git@github.com:zerocracy/farm.git'`, please restart merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#977](https://github.com/zerocracy/farm/issues/977) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two cosmetic issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda Thanks.

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**